examples/src/client/tracing_datasets.ts

```diff
@@ -101,6 +103,6 @@ what is 1213 divided by 4345?,approximately 0.2791714614499425
 
   // If using the traced dataset, you can update the datasetName to be
   // "calculator-example-dataset" or the custom name you chose.
-  const results = await client.runOnDataset(datasetName, executorFactory);
+  const results = await runOnDataset(datasetName, executorFactory, { client });
```

Comment:
I'm not such a fan of these freestanding imported functions as they have less context (also applies to all those `initialize/load` methods as well - definitely prefer the old `client.runOnDataset` personally.

Reply:
Ya it's less clean looking. Added more [context](https://github.com/hwchase17/langchainjs/pull/1549#discussion_r1219717005) in that message but the reason I initially chose this over extending the client is it also seemed slightly confusing to have an extended client

Reply:
I think this is fine, we can always mimic the old behaviour by doing `import * as client from "langchain/blabla"`